core/src/main/scala/slinky/core/package.scala

```diff
@@ -0,0 +1,73 @@
+package slinky
+
+import scala.collection.immutable.{Iterable, Queue}
+import scala.concurrent.Future
+import scala.util.Try
+
+package object core {
+
+  trait Functor[F[_]] extends Any { self =>
```

Comment:
These should not live inside a `package object`. Right now, none of the conversions will be available unless the user imports `slinky.core._`, which results in a bad dev experience. Instead, all the implicits need to live in the companion object.

Also, since this lives in Slinky and has a specific use case, we should really consider renaming `Functor` to something more specific and understandable. Maybe something like `ReactElementContainer`?

Reply:
If you look at the unit test it does't import slinky.core._ as the overriding principle was that all of this should be completely hidden. What is exposed to the user is the ReactElement.functorToFunctorElement implicit.

ReactElementContainer makes a lot more sense. Will update this.